components/bitcoin-topics/layouts/TopicBanner.tsx

```diff
@@ -245,9 +379,8 @@ export default function TopicBanner({
                                 summary={content.summary}
                             />
                             <Prose>{children}</Prose>
-                            {/* Add the GitHub edit button for mobile */}
                             <div className="max-w-3xl mx-auto mt-8 flex justify-end">
-                                <EditOnGitHubButton />
+                                <ActionButtons />
```

Comment:
Remember the discussion on `TopicBanner` and `TopicLayout`.

Topics using `TopicLayout` won't have the <ActionButtons /> component, ergo won't have the `Mark as Complete` button.

It seems only Bitcoin History will be affected by this change as it uses `TopicLayout`.

Reply:
yep, refactored now